Spring-2019/linked_lists/SOLUTIONS_2019Spring.md

```diff
@@ -89,24 +89,129 @@ If the problem is simple enough, remove this section. TODO :bug:
 
 TODO :bug:
 
-## Solutions
+Source: TODO :bug:
+
+#### Scenario
+
+Problem Statement TODO :bug:
 
-### 1. PROBLEM 1 TODO :bug:
+#### Example Input
+
+If the problem is simple enough, remove this section. TODO :bug:
+
+#### Function Signature
+
+TODO :bug:
+
+### 3. PROBLEM 2 TODO :bug:
 
 Source: TODO :bug:
 
-#### Naive/Simple Solution
+#### Scenario
+
+Problem Statement TODO :bug:
+
+#### Example Input
+
+If the problem is simple enough, remove this section. TODO :bug:
+
+#### Function Signature
 
 TODO :bug:
 
+
+## Solutions
+
+### 1. Reverse Nodes in k-Group
+
+Source: [GreeksforGreeks](https://www.geeksforgeeks.org/reverse-a-list-in-groups-of-given-size/)
```

Comment:
I mean, you can leave it as `GreeksforGreeks` if you want, but I'd change it to "GeeksForGeeks" :smile: